mnd/mnd/settings.py

```diff
@@ -19,7 +19,7 @@
 SEND_ACTIVATION_EMAIL = False
 
 PROJECT_TITLE = env.get("project_title", "MiNDAUS Registry")
-PROJECT_TITLE_LINK = "login_router"
+PROJECT_TITLE_LINK = {"viewname": "registry", "kwargs": {"registry_code": "mnd"}}
```

Comment:
Should we try to avoid hardcoding registry code, considering it is a configurable field from the UI?

Trying to think of how we could get around this, could we register a new URL that redirects to the desired page?